app/scripts/rechnungspositionen/overview/rechnungspositionenoverviewcreaterechnungen.controller.js

```diff
@@ -60,6 +60,16 @@ angular.module('openolitor-admin')
         rechnungsPositionenInWrongState: []
       };
 
+      $scope.numberOfOpenRechnungPositionen = function(rechnungPositionenIds){
```

Comment:
Sorry, this is picky, but you don't have 'rechnungPositionenIds' here but a 'rechnungPositionenList' or so.

Reply:
done